api/app/routes.py

```diff
@@ -1,6 +1,49 @@
-from app import app
+from . import app
 
+@app.route('/api/user', methods=['GET'])
+def getAllVoters():
+    return "Hello, World!"
+
+@app.route('/api/user/<uuid>', methods=['GET'])
+def getVoterInfo(uuid):
+    return "Hello, World!"
+
+@app.route('/api/election', methods=['POST'])
+def createNewElection():
+    return "Hello, World!"
+
+@app.route('/api/election', methods=['GET'])
+def getAllElections():
+    return "Hello, World!"
+
+@app.route('/api/election/<uuid>', methods=['GET'])
+def getElections(uuid):
+    return "Hello, World!"
+
+@app.route('/api/election/<uuid>', methods=['DELETE'])
+def deleteElection(uuid):
+    return "Hello, World!"
+
+@app.route('/api/election/<uuid>', methods=['PATCH'])
+def editElection(uuid):
+    return "Hello, World!"
+
+@app.route('/api/election/<uuid>/activate', methods=['PUT'])
+def activateElection(uuid):
+    return "Hello, World!"
 
-@app.route('/')
-def index():
+@app.route('/api/election/<uuid>/vote', methods=['POST'])
+def voteElection(uuid):
     return "Hello, World!"
+
+@app.route('/api/election/<uuid>/results', methods=['GET'])
+def getElectionResults(uuid):
+    return "Hello, World!"
+
+@app.route('/api/election/<uuid>/verify', methods=['GET'])
+def verifyElectionResults(uuid):
+    return "Hello, World!"
+
```

Comment:
Pretty sure you can remove this route; we're not doing blockchain/cryptographic verification stuff, just a simple authentication from the API will fix this in our MVP.